redaxo/src/core/lib/form/form_base.php

```diff
@@ -998,6 +1002,33 @@ public function getMessage()
         return $message;
     }
 
+    /**
+     * @return void
+     */
+    public function setFormAttribute(string $attributeName, ?string $attributeValue)
+    {
+        $attributeName = preg_replace('/[^\w\d\-]/', '', strtolower($attributeName));
+
+        if (0 === mb_strlen($attributeName)) {
+            return;
+        }
+
+        if (null === $attributeValue && array_key_exists($attributeName, $this->formAttributes)) {
+            unset($this->formAttributes[$attributeName]);
+            return;
+        }
+
+        if ('id' === $attributeName) {
+            $this->setFormId($attributeValue);
+        }
+
+        if (in_array($attributeName, ['method', 'action'])) {
+            return;
```

Comment:
Statt stillschweigend hier abzubrechen, lieber per Exception auf den Umstand hinweisen, dass die nicht gesetzt werden können. Irgendwie so in der Art:

```suggestion
        if (in_array($attributeName, ['method', 'action'], true)) {
            throw new rex_exception('Attribute "'.$attributeName.'" can not be set via '.__FUNCTION__);
```